renderdoc/api/replay/renderdoc_replay.h

```diff
@@ -170,7 +170,7 @@ struct IReplayOutput
   virtual ResourceId GetCustomShaderTexID() = 0;
   virtual bool PickPixel(ResourceId texID, bool customShader, uint32_t x, uint32_t y,
                          uint32_t sliceFace, uint32_t mip, uint32_t sample, PixelValue *val) = 0;
-  virtual uint32_t PickVertex(uint32_t eventID, uint32_t x, uint32_t y) = 0;
+  virtual uint32_t PickVertex(uint32_t eventID, uint32_t x, uint32_t y, uint32_t *pickedInstance) = 0;
```

Comment:
You're right sorry, when I said return I just meant as in 'pass back to the caller' and I had in mind the struct pointer method, but that's damn ambiguous :smile:.

I wouldn't worry about changing it, right now it's fine as it is. You're right though that having the bool return value would also get rid of the ~0U magic value.